code/controllers/subsystem/timer.dm

```diff
@@ -503,11 +503,17 @@ SUBSYSTEM_DEF(timer)
  * If the timed event is tracking client time, it will be added to a special bucket.
  */
 /datum/timedevent/proc/bucketJoin()
-	// Generate debug-friendly name for timer
+	// Generate debug-friendly name for timer, more or less complex/expensive depending on defines
```

Comment:
```suggestion
	// Generate debug-friendly name for timer, more or less complex/expensive depending on defines
```

should this comment go _in_ the #ifdef portion rather than sorta stick out like a sore thumb?

Reply:
it applies to both, and I didn't want to mislead the reader
I'll make one for each